spread.yaml

```diff
@@ -31,9 +36,13 @@ backends:
                 password: ubuntu
 
 exclude:
+    - .git
     - .spread-reuse.yaml
     - tests/.spread-reuse.yaml
     - $CACHE_DISABLED
+    - "*.snap"
```

Comment:
Why the quotes only on this one?  Also, where do the snap files come from?

Reply:
removed